docs/aurora/data-science/inference/vllm.md

```diff
@@ -0,0 +1,102 @@
+# Inference with vLLM
+
+vLLM is an open-source library designed to optimize the inference and serving. Originally developed at UC Berkeley's Sky Computing Lab, it has evolved into a community-driven project. The library is built around the innovative PagedAttention algorithm, which significantly improves memory management by reducing waste in Key-Value (KV) cache memory.
+
+## Install vLLM 
+
+Install vllm using pre-built wheels. 
+```bash
+module load frameworks
+conda create --name vllm python=3.10 -y
+conda activate vllm
+
+module unload oneapi/eng-compiler/2024.07.30.002
+module use /opt/aurora/24.180.3/spack/unified/0.8.0/install/modulefiles/oneapi/2024.07.30.002
+module use /soft/preview/pe/24.347.0-RC2/modulefiles
+module add oneapi/release
+
+pip install /flare/datascience/sraskar/vllm-install/wheels/*
+pip install /flare/datascience/sraskar/vllm-install/vllm-0.6.6.post2.dev28+g5dbf8545.d20250129.xpu-py3-none-any.whl
+```
+
+
+## Access Model Weights
+
+Model weights for commonly used open-weight models are downloaded and available in the following directory on Aurora:
+```bash
+/flare/datascience/model-weights/hub
+```
+To ensure your workflows utilize the preloaded model weights and datasets, update the following environment variables in your session. Some models hosted on Hugging Face may be gated, requiring additional authentication. To access these gated models, you will need a [Hugging Face authentication token](https://huggingface.co/docs/hub/en/security-tokens).
+```bash
+export HF_HOME="/flare/datascience/model-weights/hub"
+export HF_DATASETS_CACHE="/flare/datascience/model-weights/hub"
+export HF_TOKEN="YOUR_HF_TOKEN"
+```
+
+## Common Configuration Recommendations 
```

Comment:
can you give more details/examples of typical models and model sizes (in parameter counts) that typically fit in a single tile?